lib/elixir/lib/system.ex

```diff
@@ -73,11 +104,30 @@ defmodule System do
   @doc """
   Elixir build information.
 
-  Returns a keyword list with Elixir version, git short revision hash and compilation date.
+  Returns a keyword list with Elixir version, compilation date, git tag and short revision hash.
   """
   @spec build_info() :: map
   def build_info do
-    %{version: version, date: get_date, revision: get_revision}
+    %{  date:     get_date,
```

Comment:
Please fix the indentation. Also, we should leave it as `version` instead of `get_version` otherwise we will read and parse the version file twice.